service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandler.java

```diff
@@ -68,6 +71,16 @@ public void notifyAdmin(final CaseManagementOrderIssuedEvent event) {
         issuedOrderAdminNotificationHandler.notifyAdmin(event.getCaseData(), event.getCmo().getOrder(), CMO);
     }
 
+    private String getCmoOrderIssuedNotificationTemplateId(CaseData caseData) {
```

Comment:
possible we don't need urgent templates for non-CTSC users (will confirm with service)